bftengine/tests/metadataStorage/metadataStorage_test.cpp

```diff
@@ -96,10 +96,10 @@ TEST(metadataStorage_test, multi_write) {
 
 int main(int argc, char **argv) {
   ::testing::InitGoogleTest(&argc, argv);
-  KeyComparator *kk = new KeyComparator(new DBKeyComparator);
+  auto kk = make_unique<KeyComparator>(new DBKeyComparator);
```

Comment:
[Minor]: I think we can remove the `kk` variable and directly call `make_unique()` when constructing the `Client`.